### DISCUSSION TITLE
Preset ID Number on LCD

### DISCUSSION THREAD

**Discussion by azgael** [shanteacontrols/OpenDeck#201] [Ideas]
Dear Paradajz
I apologize for being a bit annoying. 
Still I would like to know if would be possible to add the preset number to the i2c display, lets say on the upper right corner. 

! - One other point is if I activate the firmware version display on start up the screen does not gets clean before starting displaying values, some characters of the first text remains in the display.

Regarding analog inputs (I guess I know the answer,"no" but anyway here it goes) is it possible to set some of them as buttons and then use them in pairs for encoders? 
Thanks 
azgael

**Comment by paradajz** [comment]
Which display are you using?

> One other point is if I activate the firmware version display on start up the screen does not gets clean before starting displaying values, some characters of the first text remains in the display.

I'll take a look at this.

> Regarding analog inputs (I guess I know the answer,"no" but anyway here it goes) is it possible to set some of them as buttons and then use them in pairs for encoders?

Please don't mix unrelated questions in a single discussion topic, but no, that is not possible. Analog inputs can only be used as buttons, not as encoders.

**Comment by azgael** [comment]
The display I am using is the SSD1306 128x64, with a raspberry pico.  Should I move this to the bug section?

I am sorry for mixing topics.

Still would it be possible to add the preset number to the i2c display, lets say on the upper right corner. I have managed to change the order of the information, I prefer the Channel number first, but my knowledge is a bit short to read the preset variable and display it. 
Thank in advance for your fast feedback!
BR
Azgael

**Comment by paradajz** [comment]
The preset number is now displayed on display.
